### ISSUE TITLE
PNotify on Nuget

### ISSUE THREAD

**Issue by AlexRebula** [sciactive/pnotify#249]
Hi. First of all I'd like to give compliments to this great plugin which I have been using now for years in my projects and it has become a must feature for all the projects I have a saying in.

That being said I'd like to add that I find it a bit odd that the last version of this plugin is NOT available on Nuget. The strange thing is that the Typescript Definitely typed package (.d.ts) is indeed available on Nuget, but the corresponding PNotify plugin version is not. That means I need to manually install the latest plugin in my ASP.NET MVC project. Which is not that bad, the only thing is that in this way I lose Nuget commodities for this plugin when updating the project, thus manually dealing with it.

So I am asking the following. Is there's any chance we will be able to see the LAST version of PNotify plugin on Nuget?

Thank you.

Regards,
Alex

**Comment by sutarmin** [comment]
This question is important for me too. It would be cool to use last PNotify in ASP.NET projects! @hperrin , may be you can tell us something about this? Or tell us who to ask about this, please.

**Comment by hperrin** [comment]
I don't actually maintain the package on Nuget. It looks to be pretty far behind. I can see if I can take it over, but I'm not sure how. I've never used Nuget.

**Comment by hperrin** [comment]
I don't really know what I'm doing. I've never used Nuget, and haven't used Visual Studio since VS6. I don't know how to make a Nuget package.

**Comment by codymullins** [comment]
I can do this one.

**Comment by delta1186** [comment]
@codymullins any update on this?  Nuget version is still 2.0.  Thanks

**Comment by codymullins** [comment]
@delta1186 the nuget repository was created by someone else, I messaged them but have not heard back. I think the only option is to create a new one which is up to @hperrin.

**Comment by delta1186** [comment]
I agree on npm. This is a older asp.net web application so NuGet fits in better in this case. NPM would be easier if I was using MVC.

**Comment by codymullins** [comment]
@delta1186 I found a [page on disputes for Nuget](https://docs.microsoft.com/en-us/nuget/policies/dispute-resolution) and have forwarded the e-mail I originally sent to the current maintainer to Nuget support. I will keep you updated.

**Comment by timavaza** [comment]
Heya, any update on this one?
+1

**Comment by codymullins** [comment]
Good news! I sent them another e-mail to follow up and they sent a reply:

```
We have sent an email to the owners of the package.

If they do not respond within 7 days, we will add you as a co-owner of the package.
```

Will follow up next week with the results.

**Comment by codymullins** [comment]
I have access and accepted ownership, should be able to get to it tomorrow or Friday at the latest - I started earlier but it's not working correctly. Note it will only be for ASP.NET framework projects and not ASP.NET Core.
